pkgs/applications/editors/vscode/update-vscodium.sh

```diff
@@ -27,7 +27,11 @@ update_vscodium () {
 # VSCodium
 
 VSCODIUM_VER=$(curl -Ls -w %{url_effective} -o /dev/null https://github.com/VSCodium/vscodium/releases/latest | awk -F'/' '{print $NF}')
-sed -i "s/version = \".*\"/version = \"${VSCODIUM_VER}\"/" "$ROOT/vscodium.nix"
+
+# TODO: update this as soon as the armv7l-linux build is fixed
+# https://github.com/VSCodium/vscodium/issues/1072
+# sed -i "s/version = \".*\"/version = \"${VSCODIUM_VER}\"/" "$ROOT/vscodium.nix"
+sed -i "s/version = if stdenv.isAarch32 then \"1.66.2\" else \".*\"/version = if stdenv.isAarch32 then \"1.66.2\" else \"${VSCODIUM_VER}\"/" "$ROOT/vscodium.nix"
```

Comment:
Why not match on `version = .*`?

Reply:
> Why not match on version = .*?

Not sure, both works for me :see_no_evil: 

```suggestion
sed -i "s/version = .*/version = if stdenv.isAarch32 then \"1.66.2\" else \"${VSCODIUM_VER}\";/" "$ROOT/vscodium.nix"
```

> (I'm assuming next versions will just have aarch32 support)

I have subscribed to all upstream activity and can manage that if needed.